general/sdk.md

```diff
@@ -22,11 +26,21 @@ directory conting the [Git](https://en.wikipedia.org/wiki/Git) repo), you
 Most files in the directory of this SDK project will be Included automatically.
 This includes configuration, text, and markdown files. It will not contain any
 `<Compile>` items unless explictly added. The SDK project is not intended to
-contain `<Compile>` items, and the binary ouptut is hidden for that reason.
+contain `<Compile>` items, and the binary output is hidden for that reason.
 
 All automatically included files and files added as `<AdditionalFiles>` are
-analyzed by the appropriate .NET Project File Analyzers: those analyzers are
-also inlcuded automatically.
+analyzed by the appropriate .NET Project File Analyzers.
+
+Those analyzers can be inlcude included with:
+
+``` XML
+<ItemGroup>
+  <PackageReference Include="DotNetProjectFile.Analyzers" Version="*" PrivateAssets="all" />
+</ItemGroup>
+```
+
+This can be in the `.net.csproj` file, but it is advised to do this in the
+ `Directory.Build.props` file instead.
 
 The SDK project can - on top of the analysis - also act as a replacement of
 the `Solution Items` folder (and other folders) a lot of `.sln` solution files
```

Comment:
something is missing here after '(and other folders)', but I don't know what you want to convey with the last part